app/decorators/physical_storage_decorator.rb

```diff
@@ -0,0 +1,23 @@
+class PhysicalStorageDecorator < MiqDecorator
+  def self.fonticon
+    'pficon pficon-container-node'
+  end
+
+  def single_quad
+    {
+      :fonticon => fonticon,
+      :tooltip  => ui_lookup(:model => type),
+    }
+  end
+
+  private
+
+  def health_state_img
```

Comment:
Unused method and also it's no longer needed, as it has been replaced by a method in QuadiconHelper.

Reply:
@skateman Just fixed it in my new push, can you take a look to see if it's correct now?